iree/compiler/Dialect/HAL/Transforms/Passes.cpp

```diff
@@ -333,13 +209,6 @@ void registerHALTransformPassPipeline() {
   PassPipelineRegistration<TransformOptions>(
       "iree-hal-transformation-pipeline",
       "Runs the full IREE HAL dialect transformation pipeline",
-      [](OpPassManager &passManager, const TransformOptions &transformOptions) {
-        buildHALTransformPassPipeline(passManager, getTargetOptionsFromFlags(),
-                                      transformOptions);
-      });
-  PassPipelineRegistration<TransformOptions>(
-      "iree-hal-transformation-pipeline2",
-      "Runs the full IREE HAL dialect transformation pipeline",
       [](OpPassManager &passManager, const TransformOptions &transformOptions) {
         buildHALTransformPassPipeline2(passManager, getTargetOptionsFromFlags(),
```

Comment:
Do you want to drop the `2` from `buildHALTransformPassPipeline2` (and the directories, etc.) in this PR?

Reply:
Can add a commit that just does the renames (it makes git absorb harder to use, but now that this is mostly stable it shouldn't matter).